components/Header/index.vue

```diff
@@ -13,7 +14,29 @@
 
 <script>
 export default {
-
+  data () {
+    return {
+      initIsXS: null
+    }
+  },
+  computed: {
+    isXS: {
```

Comment:
minor:

maybe we can rename this computed property to `isMobileViewport` ? because `isXS` and `initIsXS` kinda hard to understand for me  

just a suggestion @adzharamrullah 😄 🙏🏼